Onliner/page_objects/TVPage.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OnlinerUITest.framework;
+using OnlinerUITest.page_objects;
+using OnlinerUITest.framework;
+using OnlinerUITest.framework.elements;
+
+namespace OnlinerUITest.page_objects
+{
+    internal class TVPage : BaseOnlinerPage
+    {
+        string manufacturerCheckBoxXpath = "//ul[@class='schema-filter__list']//span[text() = '{0}']";
```

Comment:
Naming starts from UpperCase. There are not acces modifiers for these properties